cmds/nspawn

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+nspawn_list() {
+    local path=${1}
+
+    ls -1 "${path}"
+}
+
+nspawn_create() {
+    local path=${1}
+    local name=${2}
+    local file=${3}
+
+    if [ -z "${name}" ] || [ -z "${file}" ]; then
+        echo "Error: empty parameter" >&2
+        echo "create_container name Dockerfile" >&2
+        return 1
+    fi
+
+    if [ ! -e "${path}/${file}" ]; then
+        echo "No Dockerfile defined for ${file} found" >&2
+        return 1
+    fi
+
+    echo "sudo /usr/sbin/debootstrap buster $path"
+    echo "Follow the docker file and:"
+    echo "Install packages"
+    echo "Install ghc"
+    echo "Symlink /lib64/ld-linux.so.2 /lib"
+    echo "Create a user (default build)"
+    return 1
+}
+
+nspawn_doit() {
+    local name=${1}
+    local user=${2}
+    shift 2
+
+    sudo systemd-nspawn -D "$name" \
```

Comment:
`-x` as well? Just in case the build system ends up leaving things behind?

Reply:
Maybe? That would make it more docker-y. It would probably work for how I'm using it, but I should test.